src/engraving/draw/transform.cpp

```diff
@@ -427,8 +427,8 @@ Transform& Transform::rotate(double a)
         double tm23 = -sina * m_13 + cosa * m_23;
         m_13 = tm13;
         m_23 = tm23;
-        // fallthrough
     }
+    // fallthrough
```

Comment:
Seems this comment is not supposed to be inside the block in oder to be seen by MinGW (gcc)